CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# July 2, 2019
+### Rely 3.0.0
+* Added functions for performing setup and teardown operations
+  - beforeAll, afterAll, beforeEach, afterEach
+  - documentation available [here](https://reason-native.com/docs/rely/setup-teardown)
+* [**Breaking**] as a result of adding test lifecycle functions, an "env" field was added to the record passed as an argument to tests and some types gained an additional parameter. **Most users should be able to upgrade without code changes**, but utilities with manual type annotations on the test, describe, or expect types from Rely will need to be updated.
```

Comment:
Do we want to mention the additional warning 9's for this like we did previously in the changelog when adding extra fields to records?

Reply:
Good call